libraries/AP_Mount/AP_Mount_Servo.cpp

```diff
@@ -21,6 +21,7 @@ void AP_Mount_Servo::init()
         _pan_idx  = SRV_Channel::k_mount2_pan;
         _open_idx = SRV_Channel::k_mount2_open;
     }
+    _target_sysid = _params.mount_sysid_default.get();
```

Comment:
I definitely agree that we should initialise this within the init() method but ideally it should be in AP_Mount_Backend's init method.  I think we should:

1. create a default implementation of AP_Mount_Backend::init() and then have every backend (that overrides init()) also call the AP_Mount_Backend::init() as it's final action.
2. move the set_mode((enum MAV_MOUNT_MODE)_params.default_mode.get()); to AP_Mount_Backend::init()

@peterbarker, what do you think about this?